README.md

```diff
@@ -16,29 +16,35 @@ see [Indeed Hacktoberfest 2020](https://engineering.indeedblog.com/indeed-hackto
 Note: If you'd like to contribute, please either work on an issue labeled Hacktoberfest or create an issue to talk about the change you'd like to make. Spammy pull requests just to get swag are against the spirit of Hacktoberfest and won't be accepted. Remember, the purpose of Hacktoberfest is to support open source projects, not to create extra work for open source maintainers. If you're new to open source, please don't let this note discourge you though- we're happy to work with people who want to submit PRs in good faith, even if it's a small change or you're still learning :-)
 
 ### Hacktoberfest Open Office Hours
+
 Join us for open office hours to talk about issues, propose ideas, help review code, and more.
 
-| Date         | Time           | Register  |
-| ------------- |:-------------:| -----:|
+| Date                    |    Time    |                                                                                   Register |
```

Comment:
Currently, your PR can't be merged because of a merge conflict in the README. It's probably because this Hacktoberfest info is gone now. There might be other things that have changed too.
If you look at the bottom of your PR, you'll see that Github tells you it can't be merged and prompts you to resolve the conflicts.